default/stringtables/de.txt

```diff
@@ -8148,53 +8177,68 @@ BAD_DEFENSE_TROOPS_DESC
 −	Schlechte Boden[[metertype METER_TROOPS]]: 50%
 
 AVERAGE_DEFENSE_TROOPS_DESC
-'''	Mittelmäßige Boden[[metertype METER_TROOPS]]: 100%'''
+⋄	Mittelmäßige Boden[[metertype METER_TROOPS]]: 100%
 
 GOOD_DEFENSE_TROOPS_DESC
-+	Gute Boden[[metertype METER_TROOPS]]: 150%.
++	Gute Boden[[metertype METER_TROOPS]]: 150%
 
 GREAT_DEFENSE_TROOPS_DESC
-++	Hervorragende Boden[[metertype METER_TROOPS]]: 200%.
+++	Hervorragende Boden[[metertype METER_TROOPS]]: 200%
 
 ULTIMATE_DEFENSE_TROOPS_DESC
-+++	Perfekte Boden[[metertype METER_TROOPS]]: 300%.
++++	Perfekte Boden[[metertype METER_TROOPS]]: 300%
 
 BAD_DETECTION_DESC
-−	Schlechte [[metertype METER_DETECTION]]: −20 Malus.
+−	Schlechte [[metertype METER_DETECTION]]: −20 Malus
 
 GOOD_DETECTION_DESC
-+	Gute [[metertype METER_DETECTION]]: +25 Bonus.
++	Gute [[metertype METER_DETECTION]]: +25 Bonus
 
 GREAT_DETECTION_DESC
-++	Hervorragende [[metertype METER_DETECTION]]: +50 Bonus.
+++	Hervorragende [[metertype METER_DETECTION]]: +50 Bonus
 
 ULTIMATE_DETECTION_DESC
-+++	Perfekte [[metertype METER_DETECTION]]: +100 Bonus.
++++	Perfekte [[metertype METER_DETECTION]]: +100 Bonus
 
 BAD_STEALTH_DESC
-−	Schlechte [[metertype METER_STEALTH]]: −20 Malus.
+−	Schlechte [[metertype METER_STEALTH]]: −20 Malus
 
 AVERAGE_STEALTH_DESC
-'''	Mittelmäßige [[metertype METER_STEALTH]].'''
+⋄	Mittelmäßige [[metertype METER_STEALTH]]
 
 GOOD_STEALTH_DESC
-+	Gute [[metertype METER_STEALTH]]: +20 Bonus.
++	Gute [[metertype METER_STEALTH]]: +20 Bonus
 
 GREAT_STEALTH_DESC
-++	Hervorragende [[metertype METER_STEALTH]]: +40 Bonus.
+++	Hervorragende [[metertype METER_STEALTH]]: +40 Bonus
 
 ULTIMATE_STEALTH_DESC
-+++	Perfekte [[metertype METER_STEALTH]]: +60 Bonus.
++++	Perfekte [[metertype METER_STEALTH]]: +60 Bonus
 
 BAD_POPULATION_DESC
 −	Niedrige Bevölkerung 75%
 
 AVERAGE_POPULATION_DESC
-'''	Normale Bevölkerung'''
+⋄	Normale Bevölkerung
 
 GOOD_POPULATION_DESC
 +	Hohe Bevölkerung 125%
 
+GREAT_ASTEROID_INDUSTRY_DESC
++	Guter Asteroidenbergbau: + 0.2 [[metertype METER_INDUSTRY]] je [[metertype METER_POPULATION]] bei Industriefokus in Systemen mit besetzten Asteroidengürteln
+
+LIGHT_SENSITIVE_DESC
+−	Lichtempfindlich: [[metertype METER_POPULATION]] stark reduziert in Sternensystemen der Farbe [[STAR_BLUE]], weniger stark bei Sternen der Farbe [[STAR_WHITE]]
```

Comment:
Ugh, that's getting difficult here. The correct/best wording in German for this actually should be (IMO): "...stark reduziert in Systemen mit blauen Sternen, weniger stark in Systemen mit weißen Sternen".

However, if the `[[STAR_BLUE]]` and `[[STAR_WHITE]]` should/must be used, using "...Sterne der Farbe..." probably can't be avoided, as these macros get substituted by "Blau" and "Weiß" I guess?

The question is, is it necessary to use the macros here? It certainly makes the translation more awkward. @geoffthemedio?

Reply:
Translations don't need to adhere rigidly to english macro use.

Reply:
yep and all that hyperactive hyperlinking can get a little too hyped at times i agree, in this case plain german without the hyper hyper scoots better i'd say as well